.travis.yml

```diff
@@ -1,11 +1,32 @@
-language: node_js
-
-node_js:
-  - "6"
+language: objective-c
+osx_image: xcode7
+xcode_sdk: iphonesimulator9.0
 
 cache:
   directories:
     - node_modules
     - example/node_modules
+    - ~/.nvm
+
+before_install:
+  - mkdir -p ~/.nvm
+  - which nvm || [ -f ~/.nvm/nvm.sh ] || (git clone https://github.com/creationix/nvm.git ~/.nvm && cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`)
```

Comment:
Do we have to manually install node now that the language is no longer `node_js`?